Sources/Workspace/Workspace+Configuration.swift

```diff
@@ -740,90 +696,5 @@ extension Workspace {
     /// Manages a package workspace's configuration.
     // FIXME change into enum after deprecation grace period
     public final class Configuration {
-        /// The path to the mirrors file.
```

Comment:
probably can remove the whole type?

Reply:
Not without additional changes, since we add subtypes in various extensions right now, e.g. `Authorization`, `Mirrors` etc. I wanted to keep this PR focused on just removal, but restructuring this is definitely good material for a follow-up.